dm/syncer/syncer.go

```diff
@@ -2930,9 +3058,11 @@ func (s *Syncer) recordSkipSQLsLocation(ec *eventContext) error {
 }
 
 // flushJobs add a flush job and wait for all jobs finished.
+// NOTE: currently, flush job is always sync operation.
 func (s *Syncer) flushJobs() error {
```

Comment:
there's an invocation at line 1980, which is triggered by binlog heartbeat and last CheckOrSaveTime is longer than 30s by default. This flushing should also be async 🤔

Reply:
basically, the existing DM behavoir is sync flush checkpoint when it receive heartbeat. basically, we are not changing the behavoir over here, and is there any case would require us to make such change here?